apps/services/endorsements/api/src/app/modules/endorsementList/endorsementList.controller.ts

```diff
@@ -22,11 +21,12 @@ import { EndorsementListService } from './endorsementList.service'
 import { EndorsementListDto } from './dto/endorsementList.dto'
 import { FindEndorsementListByTagsDto } from './dto/findEndorsementListsByTags.dto'
 import { Endorsement } from '../endorsement/endorsement.model'
-import { BypassAuth, CurrentUser } from '@island.is/auth-nest-tools'
+import { BypassAuth, CurrentUser, Scopes } from '@island.is/auth-nest-tools'
 import { EndorsementListByIdPipe } from './pipes/endorsementListById.pipe'
 import { IsEndorsementListOwnerValidationPipe } from './pipes/isEndorsementListOwnerValidation.pipe'
 import { environment } from '../../../environments'
 import type { User } from '@island.is/auth-nest-tools'
+import { Endorsement as EndorsementScope } from '@island.is/auth/scopes'
```

Comment:
See comment below about renaming the scopes enums to end with Scope.

Reply:
Roge, will update